openstack_dashboard/static/app/core/instances/actions/delete-instance.service.js

```diff
@@ -70,7 +72,7 @@
     function initScope(newScope) {
       scope = newScope;
       context = { };
-      deletePromise = policy.ifAllowed({rules: [['instance', 'delete_instance']]});
+      deletePromise = policy.ifAllowed({rules: [['compute', 'compute:delete']]});
```

Comment:
this is correct, but now inconsistent with line 97. The allowed() function of this action is also not consistent with the server side.